### ISSUE TITLE
suggestion-bayesianoptimization can not start

### ISSUE THREAD

**Issue by leoncamel** [kubeflow/katib#206]
I tried `kubeflow` with bayesianoptimization suggestion, but it seems the image is not work.

The images I am using(same error from `cmd/suggestion/bayesianoptimization/Dockerfile`):
```text
katib            suggestionBayesianOptimizationImage 'gcr.io/kubeflow-images-public/katib/suggestion-bayesianoptimization:v0.1.2-alpha-44-gcbe5fee'
katib            suggestionGridImage                 'gcr.io/kubeflow-images-public/katib/suggestion-grid:v0.1.2-alpha-44-gcbe5fee'
katib            suggestionHyperbandImage            'gcr.io/kubeflow-images-public/katib/suggestion-hyperband:v0.1.2-alpha-44-gcbe5fee'
katib            suggestionRandomImage               'gcr.io/kubeflow-images-public/katib/suggestion-random:v0.1.2-alpha-44-gcbe5fee'
katib            vizierCoreImage                     'gcr.io/kubeflow-images-public/katib/vizier-core:v0.1.2-alpha-44-gcbe5fee'
```

The logs shows:
```text
$ kubectl logs -n kubeflow vizier-suggestion-bayesianoptimization-7578df89b6-vsszp
Traceback (most recent call last):
  File "main.py", line 6, in <module>
    from pkg.api.python import api_pb2_grpc
  File "/usr/src/app/github.com/kubeflow/katib/pkg/api/python/api_pb2_grpc.py", line 4, in <module>
    import pkg.api.python.api_pb2 as api__pb2
  File "/usr/src/app/github.com/kubeflow/katib/pkg/api/python/api_pb2.py", line 17, in <module>
    from google.api import annotations_pb2 as google_dot_api_dot_annotations__pb2
ModuleNotFoundError: No module named 'google.api'
```

**Comment by leoncamel** [comment]
@YujiOshima Any idea?

**Comment by YujiOshima** [comment]
@leoncamel I hope this is fixed by #188

**Comment by leoncamel** [comment]
@YujiOshima Cool.

Just confirmed this on [latest image](gcr.io/kubeflow-images-public/katib/suggestion-bayesianoptimization:v0.1.2-alpha-45-g3dce496), it works now.

Thanks!
